api/v1/db/oracledb/member-dao.js

```diff
@@ -166,6 +168,37 @@ const deleteMember = async (memberId) => {
   }
 };
 
+const patchMember = async (memberId, attributes) => {
+  const connection = await conn.getConnection();
+  try {
+    const memberNicknameString = `${attributes.memberNickname ? 'MEMBER_NICKNAME = :memberNickname' : ''}`;
+    const memberEmailString = `${attributes.memberEmail ? 'MEMBER_EMAIL = :memberEmail' : ''}`;
+    const memberPassword = `${attributes.memberPassword ? 'MEMBER_PASSWORD = :memberPassword' : ''}`;
+    const memberLevel = `${attributes.memberLevel ? 'MEMBER_LEVEL = :memberLevel' : ''}`;
```

Comment:
Since the logic for generating this query string is similar for each of these variables, could you refactor it?